software/receipt-wrangler.yml

```
@@ -0,0 +1,14 @@
+name: "Receipt Wrangler"
+website_url: "https://receiptwrangler.io"
+source_code_url: "https://github.com/Receipt-Wrangler"
+description: "An easy to use Receipt manager, powered by AI. Allows users to create receipts effortlesly and quickly, catagorize and more."
```

```suggestion
description: "Easy-to-use receipt manager, powered by AI. Allows users to create receipts effortlessly and quickly, categorize and more."
```

Minor syntax/typo fixes